lib/rmg/modificators/ObjectManager.cpp

```diff
@@ -447,7 +447,6 @@ bool ObjectManager::createRequiredObjects()
 					instance->object().getObjectName(), instance->getPosition(true).toString());
 				mapProxy->removeObject(&instance->object());
 			}
-			rmgNearObject.clear();
```

Comment:
If we can not clear it we should forget it somehow? What type is rmgNearObject?

Reply:
It is just `rmg::Object`. In this case, it is a resource pile adjacent to Mine and it's not supposed to ever be restored.